### PR TITLE
Add batchId and batchUrl to test result object

### PR DESCRIPTION
**Pull request by AustinNr**
Updating the result object to include additional information for downstream usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The command has the following options:
         "config": {                         // All config fields optional
           "isBlocking": true,
           "overrides":{
-
+              "startingUrl": "staging-url-override.com"
```
Comment (jjonesvermillion) [inline README.md]: maybe we want to use `example.com`
context: https://news.ycombinator.com/item?id=29623646

Comment (AustinNr) [reply README.md]: Good call! Will do

File: src/lib/__tests__/runSynthetics.test.ts [hunk 2]
```diff
@@ -58,6 +58,17 @@ describe("The runSynthetics function", () => {
     expect(startAutomatedTestsSpy).toBeCalledTimes(6);
   });
```
Comment (jjonesvermillion) [inline src/lib/__tests__/runSynthetics.test.ts:55]: May want to remove commented out code

```javascript
  //   const addMetadataToConfigSpy = jest
  //     .spyOn(gatherMetadata, "addMetadataToConfig")
  //     .mockResolvedValue(mockConfig);

```

File: src/lib/__tests__/runSynthetics.test.ts [hunk 3]
```diff
@@ -58,6 +58,17 @@ describe("The runSynthetics function", () => {
...
 
+  it("should catch and log an error that was thrown when no batch id was returned", async () => {
```
Comment (jjonesvermillion) [inline src/lib/__tests__/runSynthetics.test.ts:57]: Do you want to add a test for a fully fleshed out return object

File: src/lib/__tests__/runSynthetics.test.ts (hunk 2 above)
Comment (AustinNr) [reply src/lib/__tests__/runSynthetics.test.ts:55]: Good catch!

File: src/lib/__tests__/runSynthetics.test.ts (hunk 3 above)
Comment (AustinNr) [reply src/lib/__tests__/runSynthetics.test.ts:57]: I'll add to the other ones, good call

**Review by jjonesvermillion — APPROVED** [review]
Looks good to me

**Pull request merged**